episodes/image-display.md

```diff
@@ -0,0 +1,358 @@
+---
+title: 'Image display'
+teaching: 
+exercises: 
+---
+
+:::::::::::::::::::::::::::::::::::::: questions 
+
+- How are pixel values converted into colours for display?
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::: objectives
+
+- Create a histogram for an image
+- Install plugins from Napari Hub
+- Change colormap (LUT) in Napari
+- Adjust brightness and contrast in Napari
+- Explain the importance of always retaining a copy of the original pixel values
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+## Image array and display
+
+Last episode we saw that images are arrays of numbers with specific dimensions 
+and data type. Napari reads these numbers (pixel values) and converts them into 
+colours on our display, allowing us to view the image. Exactly how this 
+conversion is done can vary greatly, and is the topic of this episode.
+
+For example, take the image array shown below. Depending on the display 
+settings, it can look very different inside Napari:
+
+![](fig/same-array-diff-display.png
+){alt="Diagram showing an image array (top) with three different colormap 
+options (bottom)" width='70%'}
+
+Equally, image arrays with different pixel values can look the same in Napari 
+depending on the display settings:
+
+![](fig/diff-array-same-display.png
+){alt="Diagram showing two image arrays - 8-bit vs 16-bit (top) with the same 
+display (bottom)" width='50%'}
+
+In summary, we can't rely on appearance alone to understand the underlying pixel 
+values. Display settings (like the colormap, brightness and contrast - as we 
+will see below) have a big impact on how the final image looks.
+
+## Napari plugins
+
+How can we quickly assess the pixel values in an image? We could hover over 
+pixels in Napari, or print the array into Napari's console (as we saw last 
+episode), but these are hard to interpret at a glance. A much better option is 
+to use an image histogram.
+
+To do this, we will have to install a new plugin for Napari. Remember from 
+episode 1 that plugins add new features to a piece of software. Napari has 
+hundreds of plugins available on the [napari hub](https://www.napari-hub.org/) 
+website.
+
+Let's start by going to the napari hub and searching for 'matplotlib':
+
+![](fig/napari-hub.png
+){alt="Screenshot of searching 'matplotlib' on napari hub"}
+
+You should see 'napari Matplotlib' appear in the list (if not, try scrolling 
+further down the page). If we click on `napari matplotlib` this opens a summary 
+of the plugin with links to the documentation. There's also a useful 'Activity' 
```

Comment:
Useful to do a screenshot of this? 
![Uploading Screenshot 2023-12-09 at 17.05.28.png…]()

Reply:
I tried adding thee screenshot, I'm not quite sure why it didn't attach